cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsUpdateService.java

```diff
@@ -121,4 +136,43 @@ private CloudResource getCloudFormationStack(List<CloudResource> resources) {
         return resources.stream().filter(resource -> ResourceType.CLOUDFORMATION_STACK == resource.getType()).findFirst()
                 .orElseThrow(() -> new NotFoundException("CloudFormation stack is not found, the resource might have been deleted."));
     }
+
+    public void modifyVolumes(AuthenticatedContext authenticatedContext, List<String> volumeIds, String diskType, int size) throws Exception {
+        AmazonEc2Client amazonEC2Client = getEc2Client(authenticatedContext);
+        for (String volume: volumeIds) {
+            LOGGER.debug("Modifying volume {} to disk type {} and size {}", volume, diskType, size);
+            try {
+                ModifyVolumeRequest modifyVolumeRequest = ModifyVolumeRequest.builder().volumeId(volume).volumeType(diskType).size(size).build();
+                amazonEC2Client.modifyVolume(modifyVolumeRequest);
+            } catch (software.amazon.awssdk.services.ec2.model.Ec2Exception ex) {
+                LOGGER.error(String.format("AWS threw BAD Request exception: %s", ex.getMessage()));
+                if (ex.getMessage().startsWith("You've reached the maximum modification rate per volume limit.")) {
+                    continue;
```

Comment:
shouldn't we handle this usecase instead of ignoring it and continue the loop?

Reply:
anyway UT needed for this case too

Reply:
I am ignoring this because the previous flow step on the same node could have successfully modified the disk but failed to run the resize command due to some exception/ if there are multiple nodes and one node got successfully resized. The exception i am checking is specifically for that scenario. AWS throws this exception when a disk modification is already run on the disk, you can only modify a disk once in 6 hours. I will cover the unit test for this.